index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+  <link rel="stylesheet" href="mobile.css">
+</head>
+<body>
+<div class="background">
+  <header>
+    <nav>
+      <div class = "top-nav">
+        <ul class="nav-ul">
+            <li><a href="#" title="facebook"><i class="fab fa-facebook-f"></i></a></li>
+            <li><a href="#" title="twitter"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+    </div>
+    <div class="bottom-nav">
+      <div class="icon">
+        <a href="index.html"><i class="fas fa-code fa-3x"></i>
+          <h1>LL Coding Together</h1>
+        </a>
+        <ul class="links">
+          <li class="link"><a href="about.html">About</a></li>
+          <li class="link"><a href="">Program</a></li>
+          <li class="link"><a href="">Join</a></li>
+          <li class="link"><a href="">Sponsor</a></li>
+          <li class="link"><a href="">News</a></li>
+          <li class="link"><a id="border" href="">LL Coding</a></li>
+        </ul>
+      </div>
+    </div>
+    <button>
+      <i class="fas fa-bars fa-3x"></i>
+    </button>
+      <nav id="menu" class="invisible">
+        <ul class="menu-ul">
+          <li><button><i class="fal fa-times fa-3x cancel-icon"></i></button></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#About">Join</a></li>
+            <li><a href="#Contact">Sponsor</a></li>
+            <li><a href="#Contact">News</a></li>
+            <li><a href="#Contact">LL Coding</a></li>
+        </ul>
+    </nav>
+    </nav>
+  </header>
+  <main>
+    <section class="section-1">
+      <h2>"Hello World!"</h2>
+      <h2 class="h2-big">LET'S LEARN CODING TOGETHER</h2>
```

Comment:
[OPTIONAL ]  Please consider following the design by using the required color on the `h2`. This is to make sure that you follow the design guideline and replicate the design to look almost identical to the design guidelines.

 Design guideline: <img alt="Loom_Bzg357bceb" width="406" src="https://user-images.githubusercontent.com/81468675/147156690-23809f0d-335c-470e-94d2-59b153df040d.png">

 Yours: 
![ffff3](https://user-images.githubusercontent.com/57562869/148335120-dcf465b8-72ec-443a-a836-20296fd7ab99.png)